### ISSUE TITLE
tf-slim train model

### ISSUE THREAD

**Issue by fuchao01** [tensorflow/models#1263]
## Please let us know which model this issue is about (specify the top-level directory)
use tf-slim train model：
python train_image_classifier.py     --train_dir=${TRAIN_DIR}     --dataset_name=imagenet     --dataset_split_name=train     --dataset_dir=${DATASET_DIR}     --model_name=inception_v3
 raise：

Caused by op u'parallel_read/ReaderReadV2', defined at:
  File "train_image_classifier.py", line 584, in <module>
    tf.app.run()
  File "//anaconda/lib/python2.7/site-packages/tensorflow/python/platform/app.py", line 48, in run
    _sys.exit(main(_sys.argv[:1] + flags_passthrough))
  File "train_image_classifier.py", line 442, in main
    common_queue_min=10 * FLAGS.batch_size)
  File "//anaconda/lib/python2.7/site-packages/tensorflow/contrib/slim/python/slim/data/dataset_data_provider.py", line 91, in __init__
    seed=seed)
  File "//anaconda/lib/python2.7/site-packages/tensorflow/contrib/slim/python/slim/data/parallel_reader.py", line 234, in parallel_read
    reader_kwargs=reader_kwargs).read(filename_queue)
  File "//anaconda/lib/python2.7/site-packages/tensorflow/contrib/slim/python/slim/data/parallel_reader.py", line 132, in read
    enqueue_ops.append(self._common_queue.enqueue(reader.read(queue)))
  File "//anaconda/lib/python2.7/site-packages/tensorflow/python/ops/io_ops.py", line 193, in read
    return gen_io_ops._reader_read_v2(self._reader_ref, queue_ref, name=name)
  File "//anaconda/lib/python2.7/site-packages/tensorflow/python/ops/gen_io_ops.py", line 411, in _reader_read_v2
    queue_handle=queue_handle, name=name)
  File "//anaconda/lib/python2.7/site-packages/tensorflow/python/framework/op_def_library.py", line 768, in apply_op
    op_def=op_def)
  File "//anaconda/lib/python2.7/site-packages/tensorflow/python/framework/ops.py", line 2336, in create_op
    original_op=self._default_original_op, op_def=op_def)
  File "//anaconda/lib/python2.7/site-packages/tensorflow/python/framework/ops.py", line 1228, in __init__
    self._traceback = _extract_stack()

FailedPreconditionError (see above for traceback): /Users/fuchao/baidupan/imagenet/train-n09229709
	 [[Node: parallel_read/ReaderReadV2 = ReaderReadV2[_device="/job:localhost/replica:0/task:0/cpu:0"](parallel_read/TFRecordReaderV2, parallel_read/filenames)]]

**Comment by aselle** [comment]
Are you sure the data exists at the path you gave the command?

**Comment by aselle** [comment]
Please provide details about what platform you are using  (operating system, architecture). Also include your TensorFlow version. We ask for this in the issue submission template, because    it is really difficult to help without that information. Thanks!

**Comment by modanesh** [comment]
I have the same issue. I'm using macOS 10.12, python3, tensorflow 1.2.

**Comment by karmel** [comment]
Automatically closing due to lack of recent activity. Since this issue is old at this point, please reopen the issue if it still occurs when tried with the latest version of Tensorflow. Thank you.
